### ISSUE TITLE
Support for Docker Compose

### ISSUE THREAD

**Issue by almereyda** [dokku-alt/dokku-alt#208]
Now that **fig** has finally been migrated to [**Docker Compose**](https://github.com/docker/docker/issues/9694), it would be nice to see official support here.

It could nicely integrate with buildpacks, somehow.

Until now, I can only imagine to layer Docker Compose inside a container I would be pushing, not a desired behaviour.

**Comment by ayufan** [comment]
Yes. I saw. It looks pretty nice. I don't know if you saw the https://github.com/project-nsmg/figure. Maybe we could somehow integrate it?

**Comment by almereyda** [comment]
I didn't know that one, no. Maybe for the start it could we enough to run figure inside a Docker container and go from there? Else I'm kind of getting distracted from all these Dokku forks. Therefore #146 is really important.

**Comment by mc0e** [comment]
I note also that the format for environment variables used by compose is different, eg in the case of things like database connections:  http://docs.docker.com/compose/env/

With people distributing docker recipes that expect this format, it would be nice to be able to get dokku-alt to provide the environment variables in the expected format for database services, so that third party docker recipes can be used unmodified.

e.g. I'm currently trying to get https://github.com/sameersbn/docker-redmine to run on a dokku-alt server, and this is my current stumbling block.  

    git clone git@github.com:sameersbn/docker-redmine.git
    cd docker-redmine
    git remote add dokku@myhost.com:redmine
    dokku create redmine
    dokku volume:create redmine /home/redmine/data
    dokku volume:link redmine redmine
    dokku mariadb:create redmine
    dokku mariadb:link redmine redmine

It'd be nice if that was all, but in order to get the container to run I now have to locate the DATABASE_URL value being provided and translate it to composer style variables and set them with `dokku config`.  It's not the end of the world, but it would be nice to have a dokku-alt command that does it for me, or (better) a way to tell the mariadb, postgres, redis, etc plugins to provide the connection data in Compose style (which might gradually become the default over time).
